gpt4all-chat/embllm.cpp

```diff
@@ -303,6 +303,20 @@ EmbeddingLLM::~EmbeddingLLM()
     m_embeddingWorker = nullptr;
 }
 
+QString EmbeddingLLM::model() const
+{
+    const EmbeddingModels *embeddingModels = ModelList::globalInstance()->installedEmbeddingModels();
+    if (!embeddingModels->count())
+        return QString("Unknown");
+
+    const ModelInfo defaultModel = embeddingModels->defaultModelInfo();
+    if (!defaultModel.name().isEmpty())
+        return defaultModel.name();
+    else if (!defaultModel.id().isEmpty())
+        return defaultModel.id();
+    return defaultModel.filename();
```

Comment:
Since we only intend to support a fixed set of embedding models, I don't like the fallback to the id/filename here. We should pick a property that works for all of the embedding models we will provide.

Reply:
fixed.